Make it work!

BachToTheFuture opened the pull request:
@scottmc @humdingerb @owenca
ZooKeeper should work now 😃

BachToTheFuture committed:
Make it work!

BachToTheFuture committed:
Actually fix the build

BachToTheFuture committed:
Add instructions

scottmc commented:
While you are at it, bump version to 2.1.1, as that's what the next tagged release should be.  Be sure to run it through a few tests to make sure it works as expected.  I see a sample script, does that function as expected?

BachToTheFuture committed:
Bump to version 2.1.1

BachToTheFuture committed:
Bump to 2.1.1

bach5000 commented:
@scottmc I ran the sample test script and it works as expected.
I had to fix some things, though. I'm updating the PR right now.

bach5000 commented:
@scottmc I ran the sample test script and it works as expected.
I had to fix some things, though. I'm updating the PR right now.

bach5000 commented:
Never mind... found `chmod -x` should do it.

BachToTheFuture commented:
Added some things with the new commit:
* the examples now run
* updated instructions in README
* toggling "In Terminal" and "Keep Open" will show a helpful log for debugging
* Successfully run commands show "Success" while those that fail will show "Command failed to run"
